src/appmixer/google/gmail/package.json

```diff
@@ -1,7 +1,8 @@
 {
-    "name": "appmixer.goog.gmail",
+    "name": "appmixer.google.gmail",
     "version": "1.0.0",
     "dependencies": {
+        "axios": "^1.7.7",
```

Comment:
Use `context.httpRequest` instead of `axios`. This way you don't need to install any dependencies. `context.httpRequest` is the same as `axios`.

Reply:
@jirihofman thank you! I have made the changes